src/renderer/gl_engine/tvgGlRenderer.cpp

```diff
@@ -1281,6 +1281,10 @@ RenderData GlRenderer::prepare(RenderSurface* image, RenderData data, const Matr
         sdata->texId = _genTexture(image);
         sdata->opacity = opacity;
         sdata->texColorSpace = image->cs;
+        if (!image->premultiplied) {
+            if (image->cs == ColorSpace::ABGR8888) sdata->texColorSpace = ColorSpace::ABGR8888S;
```

Comment:
Hmm... I think the image loader might need to assign the proper colorspace value.